src/track/external-captions-handler.js

```diff
@@ -231,6 +231,15 @@ class ExternalCaptionsHandler extends FakeEventTarget {
     }
   }
 
+  addCueChangeListener(textTrackEl: TextTrack): void {
+    this._eventManager.listen(textTrackEl, 'cuechange', (e: FakeEvent) => this._onCueChange(e));
+  }
+
+  _onCueChange(e: FakeEvent): void {
```

Comment:
pls see is if you can use this method also here 
https://github.com/kaltura/playkit-js/blob/v0.72.0/src/track/external-captions-handler.js#L382
anyway need to verify that the active cues there are normalized

Reply:
the two method called in two different cases, the other method you talk about called only in case of using costume caption (look at lines 216/219)